spec/twilio_adapter_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'twilio_adapter'
+
+describe TwilioAdapter do
+  
+  it { is_expected.to respond_to :send_message }
+
+end
```

Comment:
could maybe try and apply doubles and stubs here for more tests, although I struggled with deciding on the best way to go about this and not have vacuous tests